tests/utils_tests/test_dateparse.py

```diff
@@ -80,6 +80,23 @@ def test_parse_python_format(self):
         for delta in timedeltas:
             self.assertEqual(parse_duration(format(delta)), delta)
 
+    def test_parse_postgres_format(self):
+        self.assertEqual(parse_duration('1 day'), timedelta(1))
```

Comment:
Oh, nice. First time I've had a chance to use subTest.  I'll create a PR for making the same change in another test in this same function.